src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -123,38 +132,45 @@ const sl_wifi_device_configuration_t config = {
     .mac_address = NULL,
     .band        = SL_SI91X_WIFI_BAND_2_4GHZ,
     .region_code = REGION_CODE,
-    .boot_config = { .oper_mode = SL_SI91X_CLIENT_MODE,
-                     .coex_mode = SL_SI91X_WLAN_BLE_MODE,
-                     .feature_bit_map =
-#ifdef SLI_SI91X_MCU_INTERFACE
-                         (SL_SI91X_FEAT_SECURITY_OPEN | SL_SI91X_FEAT_WPS_DISABLE),
-#else
-                         (SL_SI91X_FEAT_SECURITY_OPEN | SL_SI91X_FEAT_AGGREGATION | SL_SI91X_FEAT_ULP_GPIO_BASED_HANDSHAKE |
-                          SL_SI91X_FEAT_DEV_TO_HOST_ULP_GPIO_1),
+    .boot_config = { .oper_mode       = SL_SI91X_CLIENT_MODE,
```

Comment:
If would be good to have a base` sl_wifi_device_configuration_t config`
with everything that is common already set and removing all ifdefs from that declaration.

Then when config is used in `sl_matter_wifi_platform_init` have the conditional statement update the needed fields

Reply:
Since, this is not a part of this PR. I will open a ticket to create a builder approach for the configuration.